code/modules/organs/internal/_internal.dm

```diff
@@ -3,10 +3,12 @@
 ****************************************************/
 /obj/item/organ/internal
 	scale_max_damage_to_species_health = TRUE
+	var/tmp/prosthetic_name //Name given to the organ when robotized. Null means no changes
 	var/tmp/alive_icon //icon to use when the organ is alive
 	var/tmp/dead_icon // Icon to use when the organ has died.
 	var/tmp/prosthetic_icon //Icon to use when the organ is robotic
 	var/tmp/prosthetic_dead_icon //Icon to use when the organ is robotic and dead
+	var/tmp/prosthetic_icon_file //The path to the icon file for the prosthetic icons. Null means use the current 'icon' value.
```

Comment:
Not a fan of every organ in the game having these vars defined on them - they should be handled by a prosthetics model, defaulting to the base type if none is provided.

Reply:
Internal organs have no prosthetic models. 
Also, adding prosthetic models to internal organs is like a brand new feature, and its out of scope for a fix PR. I probably could do it, but not on this pr.